docs/README

```diff
@@ -147,6 +147,26 @@ meaning that it is possible to unmap any aligned area and turn it into a hole.
 This is implemented using the "FALLOC_FL_PUNCH_HOLE" "mode" of the
 "fallocate()" system call.
 
+Note: If running on the ZFS file system, the default module parameters
```

Comment:
It seems you've put this under the "Concept" section as a "Note". May be there is a better place for it, may be you could create a separate section for quirks like this, outside of the "Concept" section?

Reply:
The concept deals directly with technical details like  FIEMAP/SEEK_HOLE etc which is the source of the problem caused by the zfs configuration, which is why i was trying to keep it related. However, I can appreciate how it would be better to put it in its own section, in case their are others in future. I'll get the change sorted :+1: 


etc i was trying to relate the zfs issue to it.